csfieldguide/templates/interactives/base/iframe.html

```diff
@@ -26,6 +26,6 @@
     {% block js %}
     {% endblock js %}
 
-    <script src="{% static 'js/third-party/iframeResizer.contentWindow.min.js' %}"></script>
+    <script src="{% static 'js/iframe-resizer-content-window.js' %}"></script>
```

Comment:
This `script` tag is missing the `type="text/javascript"` attribute, though it's not needed in HTML5. However it seems every other `script` tag has it in this project. Should we add it here?